# License identified for kernel/sched/MuQSS.c:
// SPDX-License-Identifier: GPL-2.0
/*
 *  kernel/sched/MuQSS.c, was kernel/sched.c
 *
 *  Kernel scheduler and related syscalls
 *
 *  Copyright (C) 1991-2002  Linus Torvalds
 *

--- a/kernel/sched/MuQSS.c.orig2	2023-12-13 01:13:08.876928681 -0800
+++ b/kernel/sched/MuQSS.c	2023-12-13 01:08:58.844930144 -0800
@@ -66,7 +66,7 @@
 #include <asm/tlb.h>
 
 #include "../workqueue_internal.h"
-#include "../../fs/io-wq.h"
+#include "../../io_uring/io-wq.h"
 #include "../smpboot.h"
 
 #include "MuQSS.h"
